packages/template-ui/src/components/mixins/headerMixin.js

```diff
@@ -27,8 +27,5 @@ export default {
     headerImageURL() {
       return this.sectionImageURL || this.getAssetURL('headerImage');
     },
-    hasHeaderImage() {
```

Comment:
This method is also used in the `ChannelNavBar` component, maybe the same style should be removed there too.

https://github.com/endlessm/kolibri-explore-plugin/blob/T32923-back-button/packages/template-ui/src/components/ChannelNavBar.vue#L5

Reply:
I remove it in the other PR. See #310